### ISSUE TITLE
"Tournmants disabled due to rooted device"

### ISSUE THREAD

**Issue by reszkojr** [efroemling/ballistica#194]
I am trying to play tournmants on bombsquad, but i can't, cause It says that my device is rooted, and It is not, i even installed root checker and it says that i'm not rooted

**Comment by efroemling** [comment]
Sorry about that; I've got a root-checker built into the Android version and sometimes it gets false positives. I'm working towards being able to run tournaments on my own servers and then I can finally disable those checks.

**Comment by SushrutShidhaye** [comment]
Same
But my devices was rooted o_o
So my account got banned...:(
So now i cant unlink any linked account it shows try after 99999 daysss....
I juat lost my Bombsquad Pro...:(

**Comment by Okzser** [comment]
Yeah dude, but my device isn't rooted, i can't freacking play tournmants
anymore


Em qui, 22 de out de 2020 05:45, SushrutShidhaye <notifications@github.com>
escreveu:

**Comment by Dliwk** [comment]
And why was this check needed at all if anyone can play tournaments from a "rooted" PC? :thinking:

**Comment by lemonyPRIDE** [comment]
@Dliwk maybe because  your "ROOTED" PC doesnt require those checks, because everything in a pc except for bios is available at C drive, and the user can literally customize windows according to his/her own desire. So its kind of pointless to make those root-checker. 🤷‍♂️

**Comment by Dliwk** [comment]
Well, I say, PC players have full access to game files.

**Comment by baudav** [comment]
Hi,
same problem on device Realme 8i, not rooted but notify disabled due to rooted device.

**Comment by Loup-Garou911XD** [comment]
Closing as outdated
